#goal definition
float64 shelf_direction
---
#result definition
bool success
---
#feedback
